metricflow/test/integration/test_cases/itest_simple.yaml

```diff
@@ -13,6 +13,43 @@ integration_test:
     GROUP BY
       is_instant
 ---
+integration_test:
+  name: simple_query_with_no_group_bys
+  description: Tests selecting a metric with no group bys.
+  model: SIMPLE_MODEL
+  metrics: ["booking_value"]
```

Comment:
Can you try this on a ratio metric requiring a join? Something like `bookings_per_listing` should do it. Last time I tried it generated invalid SQL, which means users will hit failure states with difficult to understand SQL syntax errors.

Reply:
oo yea this breaks on ratio metrics requiring a join.